### PR TITLE
refac: add subsystem test

### PR DESCRIPTION
**Pull request by johevemi**
Added subsystem test to verify that each run's calculation version number has increased.

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [x] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/dotnet/subsystem-tests/Features/Calculations/WholesaleFixingCalculationScenario.cs [hunk 1]
```diff
@@ -31,9 +31,31 @@ namespace Energinet.DataHub.Wholesale.SubsystemTests.Features.Calculations;
...
+        var (calculationVersion, message) = await Fixture.GetLatestCalculationVersionFromCalculationsAsync();
+
+        // Assert
+        if (calculationVersion == string.Empty)
+        {
+            throw new InvalidOperationException(message);
+        }
+
+        // Set the property value
+        CalculationVersion = calculationVersion;
```
Comment (energinet-ajw) [inline source/dotnet/subsystem-tests/Features/Calculations/WholesaleFixingCalculationScenario.cs:58]: Maybe the assert could be a value larger than 0.

File: source/dotnet/subsystem-tests/Features/Calculations/Fixtures/CalculationScenarioFixture.cs [hunk 2]
```diff
@@ -329,6 +329,57 @@ public async Task<Response<LogsQueryResult>> QueryLogAnalyticsAsync(string query
...
+    public async Task<(string CalculationVersion, string Message)> GetCalculationVersionOfCalculationIdFromCalculationsAsync(
+        string calculationId)
+    {
+        try
+        {
+            var statement = DatabricksStatement.FromRawSql(
+                $"SELECT calculation_version FROM {Configuration.DatabricksCatalogName}.wholesale_internal.calculations WHERE calculation_id = '{calculationId}'");
+            var queryResult = DatabricksSqlWarehouseQueryExecutor.ExecuteStatementAsync(statement.Build());
+            var list = await queryResult.ToListAsync();
+
```
Comment (energinet-ajw) [inline source/dotnet/subsystem-tests/Features/Calculations/Fixtures/CalculationScenarioFixture.cs:341]: How about just returning list.First?

File: source/dotnet/subsystem-tests/Features/Calculations/WholesaleFixingCalculationScenario.cs [hunk 3]
```diff
@@ -31,9 +31,31 @@ namespace Energinet.DataHub.Wholesale.SubsystemTests.Features.Calculations;
...
-    public WholesaleFixingCalculationScenario(LazyFixtureFactory<CalculationScenarioFixture> lazyFixtureFactory)
+    public WholesaleFixingCalculationScenario(LazyFixtureFactory<CalculationScenarioFixture> lazyFixtureFactory, string calculationVersion)
         : base(lazyFixtureFactory)
     {
+        CalculationVersion = calculationVersion;
+    }
+
+    public string CalculationVersion { get; private set; }
+
+    [ScenarioStep(-1)]
```
Comment (energinet-ajw) [inline source/dotnet/subsystem-tests/Features/Calculations/WholesaleFixingCalculationScenario.cs:42]: Does -1 work?

File: source/dotnet/subsystem-tests/Features/Calculations/Fixtures/CalculationScenarioFixture.cs (hunk 2 above)
Comment (johevemi) [reply source/dotnet/subsystem-tests/Features/Calculations/Fixtures/CalculationScenarioFixture.cs:341]: sure

File: source/dotnet/subsystem-tests/Features/Calculations/WholesaleFixingCalculationScenario.cs [hunk 4]
```diff
@@ -268,4 +290,22 @@ public async Task AndThen_CalculationOrchestrationIsCompleted()
...
+        // Act
+        var (calculationVersion, message) = await Fixture.GetCalculationVersionOfCalculationIdFromCalculationsAsync(Fixture.ScenarioState.CalculationId.ToString());
+
+        // Assert
+        if (calculationVersion == string.Empty)
+        {
+            throw new InvalidOperationException(message);
+        }
+
+        Assert.True(Convert.ToInt32(calculationVersion) > Convert.ToInt32(CalculationVersion));
```
[the remark this answers is not in the document]
Comment (johevemi) [reply source/dotnet/subsystem-tests/Features/Calculations/WholesaleFixingCalculationScenario.cs]: ok

File: source/dotnet/subsystem-tests/Features/Calculations/WholesaleFixingCalculationScenario.cs (hunk 1 above)
Comment (johevemi) [reply source/dotnet/subsystem-tests/Features/Calculations/WholesaleFixingCalculationScenario.cs]: yes

**Review by energinet-ajw — APPROVED** [review]

**Pull request merged**